lib/connectors_sdk/base/http_call_wrapper.rb

```diff
@@ -99,8 +99,24 @@ def service_type
         self.class::SERVICE_TYPE
       end
 
+      def describe
+        {
+          :name => name,
+          :connection_requires_redirect => connection_requires_redirect,
+          :configurable_fields => configurable_fields
+        }
+      end
+
       private
 
+      def connection_requires_redirect
+        false
```

Comment:
Interestingly, no Crest connector uses OAuth and as far as I could understand we'd also love to get rid of OAuth, I think that's a good default value

Reply:
I think of the OAuth handshake as a special case, even if the majority of our internal connectors use it. Mostly because there's no _alternative_ flow to the OAuth handshake, we simply don't do it if it's not needed. Leaving it as defaulted to _false_ is a more natural default as connectors should opt into the redirect behavior.

And as Artem alludes to, I expect it to be uncommon for _new_ connectors.